### PR TITLE
Added spanish translation

### PR DESCRIPTION
**Pull request by salvadorp2001**
However, I was not able to test it because when trying to compile, it complains about
```config.status: error: cannot find input file: `po/Makefile.in.in'```

### CODE REVIEW

**Comment by 12345ieee** [comment]
Really? Have you run:
```
./autogen.sh && ./configure --enable-gui && make
```
with the needed dependencies installed?

File: po/es.po [hunk 1]
```diff
@@ -0,0 +1,530 @@
+# Spanish translations for Game Conqueror.
+# Copyright (C) 2009-2014 Wang Lu <coolwanglu(a)gmail.com>
+# This file is distributed under the same license as Game Conqueror.
+# Salvador Pardiñas <darkfm(at)vera.com.uy>, 2014.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: scanmem 0.14\n"
```
Comment (12345ieee) [inline po/es.po]: The metadata here are a bit messed up, current version is v0.16 and the revision date of this translation sure isn't in 2014.

**Comment by salvadorp2001** [comment]
D'oh. I copied the metadata at the top from another translation and forgot to change dates.

**Comment by 12345ieee** [comment]
The file looks ok now.
I'm still a little puzzled by your error message, are you sure `autogen.sh` runs without errors?

**Comment by salvadorp2001** [comment]
I'd forgotten to run autogen, then forgotten that Arch Linux doesn't install intltool by default. Then an error about not having PyGObject, then an error about "C Library doesn't support this locale". After fixing all that, yes, it seems to work and seems to be correct.

**Comment by 12345ieee** [comment]
Happy to hear it worked.

I'll merge it later, thanks for contributing!

**Comment by 12345ieee** [comment]
Merged as 328a7c6eab2c9c8dd .
Thanks for the translation.

Closing.

**Pull request closed without merging**